docs/tutorials/writing-documentation.md

```diff
@@ -49,12 +52,15 @@ MkDocs detects file changes in the documentation, rebuild it and reload the brow
 
 There are two main configuration files:
 
-- `.readthedocs.yaml`: configuration for the service [Read the Docs](https://readthedocs.org/) to know how to build the documentation with MkDocs
-- `.mkdocs.yaml`: configuration of [MkDocs](https://www.mkdocs.org/) for the documentation (navigation, extensions, theme, etc)
+- `.readthedocs.yaml`: configuration for the service [Read the Docs](https://readthedocs.org/) to know how to build the
+  documentation with MkDocs
+- `.mkdocs.yaml`: configuration of [MkDocs](https://www.mkdocs.org/) for the documentation (navigation, extensions,
+  theme, etc)
```

Comment:
idem, probably autowrapping ?

Reply:
yes, and it is to be able to read the markdown with a text editor